src/dotnet/commands/dotnet-new/CSharp_Lib/Library.cs

```diff
@@ -4,8 +4,8 @@ namespace ClassLibrary
 {
     public class Class1
     {
-        public void Method1() {
-            
+        public void Method1()
+        {    
         }
 
```

Comment:
Yes, we don't have those unnecessary blank lines in the console app's Program.cs or in xunit's Tests.cs.

```C#
using System;

namespace ConsoleApplication
{
    public class Program
    {
        public static void Main(string[] args)
        {
            Console.WriteLine("Hello World!");
        }
    }
}
```

```C#
using System;
using Xunit;

namespace Tests
{
    public class Tests
    {
        [Fact]
        public void Test1() 
        {
            Assert.True(true);
        }
    }
}
```

```C#
using System;

namespace ClassLibrary
{
    public class Class1
    {
        public void Method1()
        {    
        }

    }
}
```

See the inconsistency?